src/cljs/main/broadfcui/page/workspace/method_configs/method_config_editor.cljs

```diff
@@ -41,12 +41,12 @@
              error (style/create-server-error-message error)
              :else [comps/Spinner {:text "Loading details..."}])))
    :component-did-mount
-   (fn [{:keys [this]}]
-     (this :load-agora-method))
+   (fn [{:keys [this props state]}]
+     (react/call :load-agora-method this props state))
```

Comment:
There's no reason to pass React fields to other methods.  This is passing all of `props`, and `:load-agora-method` is calling it `method-ref`.  And the `state` parameter is being dropped on the floor.